src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -531,6 +531,9 @@ void VtolAttitudeControl::task_main()
 			// update transition state if got any new data
 			if (got_new_data) {
 				_vtol_type->update_transition_state();
+
+				_vtol_type->process_mc_data();
+				fill_mc_att_rates_sp();
```

Comment:
I'm quite comfortable having Simon drive this. The transition / multi-mode publication code has been buggy for months now and I think some software engineering experience combined with the intense testing will help a lot. In particular since you have already two VTOL's to test with.